### ISSUE TITLE
Polygon obscuring map

### ISSUE THREAD

**Issue by osesina** [codeforboston/cambridge_voting_locations#80]
When obtaining directions for election day voting, the precinct polygon obscures visuals on the map.
Do we need to show the polygon at all?

![image](https://cloud.githubusercontent.com/assets/2693547/19857421/f5554c92-9f53-11e6-901d-feee442f53d5.png)

**Comment by moczka** [comment]
yeah the polygon is necessary because it highlights the precinct. My map is well centered and with the right scale. It doesn't scale well when I have dev tools.
